doc/reference/tooling/tt_cli/replicaset.rst

```diff
@@ -0,0 +1,167 @@
+.. _tt-replicaset:
+
+Working with replicasets
+=========================
+
+..  code-block:: console
+
+    $ tt replicaset COMMAND {APPLICATION[:APP_INSTANCE] | URI} [OPTIONS ...]
+    # or
+    $ tt rs COMMAND {APPLICATION[:APP_INSTANCE] | URI} [OPTIONS ...]
+
+``tt replicaset`` (or ``tt rs``) manages a Tarantool replica set.
+
+``COMMAND`` is one of the following:
+
+*   :ref:`status <tt-replicaset-status>`
+*   :ref:`promote <tt-replicaset-promote>`
+*   :ref:`demote <tt-replicaset-demote>`
+
+
+.. _tt-replicaset-status:
+
+status
+------
+
+..  code-block:: console
+
+    $ tt replicaset status {APPLICATION[:APP_INSTANCE] | URI} [OPTIONS ...]
+    # or
+    $ tt rs status  {APPLICATION[:APP_INSTANCE] | URI} [OPTIONS ...]
+
+``tt replicaset status`` (``tt rs status``) shows the current status of a replica set.
+
+
+.. _tt-replicaset-status-application:
+
+Displaying status of all replica sets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To view the status of all replica sets of an application in the current ``tt``
+environment, run ``tt replicaset status`` with the application name:
+
+..  code-block:: console
+
+    $ tt replicaset status myapp
+
+.. _tt-replicaset-status-instance:
+
+Displaying status of a single replica set
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To view the status of a single replica set of an application , run ``tt replicaset status``
+with a name or a URI of an instance from this replica set:
+
+..  code-block:: console
+
+    $ tt replicaset status myapp:storage-001-a
+
+..  code-block:: console
+
+    $ tt replicaset status 192.168.10.10:3301
```

Comment:
Am I correct that displaying a status of a remote replica set (when URI is passed) always requires authentication? Maybe, it would be better to make a separate intro for this code snippet and mention the privileges required to execute the command.